### ISSUE TITLE
Color mode setting under User Preferences does not work

### ISSUE THREAD

**Issue by pv2b** [netbox-community/netbox#13760]
### NetBox version

v3.6.1

### Python version

3.11

### Steps to Reproduce

1. Click on your username in top right corner
2. Click on Preferences in the profile menu
3. Change the "Color mode" setting, to something new, and click save.

### Expected Behavior

Color mode changes to desired setting (dark or light)

### Observed Behavior

Colour mode is not changed

Workaround: Use dark mode/light mode toggle in the profile menu instead.

**Comment by pv2b** [comment]
Hi! Are you sure this is a duplicate?

I found the report you mentioend before I submitted my own, but that is in relation to the "system theme" changing. I interpreted this to mean that this is in the case of changing the "theme" of your operating system which can interact with web browsers and preference for dark/light mode.

This report is for a different thing, regarding changing user preferences, not system preferences.
